tasks/windows.ps1

```diff
@@ -53,10 +53,12 @@ function Get-HostName
 function Get-CA($Master)
 {
   $verificationCallback = [Net.ServicePointManager]::ServerCertificateValidationCallback
+  $preservedProtocol = [Net.ServicePointManager]::SecurityProtocol
   try
   {
     # temporarily disable SSL verification while downloading CA
     [Net.ServicePointManager]::ServerCertificateValidationCallback = { $true }
+    [Net.ServicePointManager]::SecurityProtocol = [Net.ServicePointManager]::SecurityProtocol -bor [Net.SecurityProtocolType]3072
```

Comment:
I'd like to see this constant commented in the code, rather than just in the commit message - customers do look at this stuff on-disk sometimes.

Reply:
done